### ISSUE TITLE
Maybe new conflicts

### ISSUE THREAD

**Issue by LLinVIP** [unocss-applet/unocss-applet#6]
Excellent work, compared with 'unocss-preset-uni', maybe some giant changes happened in this tool.

However, there is a question. Before I using  'unocss-applet', the compile process could go on smoothly, after using this, an error occoured. This is screenshot:

![image](https://user-images.githubusercontent.com/68938689/187482878-122c237d-b1c8-4383-b0e5-72934b3d5ec6.png)

tm-sheet is a component which I using.

![image](https://user-images.githubusercontent.com/68938689/187483289-47e031be-ffe1-4c03-bf74-2c032b91550d.png)
After deleting line 24, vite could pass, but weChat miniprogram tool occured error:

![image](https://user-images.githubusercontent.com/68938689/187484024-2e5f249d-7752-46d2-a0df-4e69d411bf23.png)
![image](https://user-images.githubusercontent.com/68938689/187484035-c0376170-8b8e-4da3-a652-ff68e29a9ada.png)

I think there may be some problems.

By the way, an another question is does unocss has the configuration which could set individual prefix like Tailwind CSS

![image](https://user-images.githubusercontent.com/68938689/187484784-9bd59e19-d4be-42f3-b1bb-9cf581646f25.png)

it may solve the conflicts between components and Unocss.

Thanks!

**Comment by liwenfengg** [comment]
You can use this version ` "UnoCSS-applet": "^0.1.2" first`

**Comment by zguolee** [comment]
We fixed `whitespace was expected` in `v0.2.5`, because the `transformer-attributify` make mistakes. However, there are still problems, even if only `UnoCSS` is used.

```
vite v2.9.15 building for development...
[plugin:vite:vue] [@vue/compiler-sfc] Unexpected token (71:8)

/Users/zguo/Downloads/tmui/src/tmui/components/tm-checkbox/tm-checkbox.vue
114|      beforChecked:{
115|          type:[Function,String,Boolean],
116|          =>{
   |          ^
117|              return false;
118|          }
at tmui/components/tm-checkbox/tm-checkbox.vue:71:8
```

**Comment by zguolee** [comment]
> By the way, an another question is does unocss has the configuration which could set individual prefix like Tailwind CSS
> 

It's provided. You can check [PresetMiniOptions](https://github.com/unocss/unocss/blob/main/packages/preset-mini/src/index.ts#L30-L55), and `PresetAppletOptions` extends `PresetMiniOptions`.

**Comment by LLinVIP** [comment]
> > By the way, an another question is does unocss has the configuration which could set individual prefix like Tailwind CSS
> 
> It's provided. You can check [PresetMiniOptions](https://github.com/unocss/unocss/blob/main/packages/preset-mini/src/index.ts#L30-L55), and `PresetAppletOptions` extends `PresetMiniOptions`.


Thanks. After adding this item, it works fine and prevent some conflicts.
![image](https://user-images.githubusercontent.com/68938689/187630812-b3817179-a5d7-4386-9965-1c25f752ea81.png)

**Comment by LLinVIP** [comment]
> We fixed `whitespace was expected` in `v0.2.5`, because the `transformer-attributify` make mistakes. However, there are still problems, even if only `UnoCSS` is used for [TMUI](https://gitee.com/LYTB/tmui-design).
> 
> ```
> vite v2.9.15 building for development...
> [plugin:vite:vue] [@vue/compiler-sfc] Unexpected token (71:8)
> 
> /Users/zguo/Downloads/tmui/src/tmui/components/tm-checkbox/tm-checkbox.vue
> 114|      beforChecked:{
> 115|          type:[Function,String,Boolean],
> 116|          =>{
>    |          ^
> 117|              return false;
> 118|          }
> at tmui/components/tm-checkbox/tm-checkbox.vue:71:8
> ```

We need a little change in the code like this:
![image](https://user-images.githubusercontent.com/68938689/187635665-4bd70e72-ae3d-40f6-8687-7eec8b21a61a.png)

It will compile well.
